workflows/genomeassembler.nf

```diff
@@ -40,8 +40,53 @@ ch_multiqc_custom_methods_description = params.multiqc_methods_description ? fil
 // SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
 //
 // include { INPUT_CHECK } from '../subworkflows/local/input_check'
-include { PREPARE_INPUT } from '../subworkflows/local/prepare_input'
+include { PREPARE_INPUT } from "$projectDir/subworkflows/local/prepare_input"
```

Comment:
I just prefer this format of using paths relative to `$projectDir` rather than relative to current workflow. Some others prefer this too, but it's not been implemented in general because nf-core tests need to use `$launchDir` instead, and so they stuck to using relative paths. These implicit variables are not too well understood.

Reply:
I have no preference as long as it works.
I tested `nextflow run mahesh-panchal/genomeassembler -r quality_check_workflows -profile test,singularity --outdir results` locally and it worked. So ok to me.